Example/Example/Model/Model.swift

```diff
@@ -1,7 +1,7 @@
 import Faro
 
 /// Example model
-class Model: Parseable {
+class Model: Deserializable {
```

Comment:
Parseable/mappable zoals het voordien (kind of) was dan? Al is Mappable minder duidelijk dan Parseable qua intentie aangeven.